periodic/periodic.go

```diff
@@ -575,3 +579,34 @@ func (r *RunnerResults) ID() string {
 	}
 	return base
 }
+
+//TODO remove above if condition when receive and sent byte count is added to GRPC as well.
+// String prints the Runner Results
+func (r *RunnerResults) String() string {
+	if r.SentRequestSizeKBPS < 0.01 && r.ReceivedResponseSizeKPBS < 0.01 {
+		return fmt.Sprintf("All done %d calls %.3f ms avg, %.1f qps,\n",
```

Comment:
coma and \n is probably wrong

also just add to the common string instead of duplicating the whole common part